dist/rpm/os-autoinst.spec

```diff
@@ -181,6 +181,8 @@ done
 rm xt/00-tidy.t
 # Remove test relying on a git working copy
 rm xt/30-make.t
+# https://progress.opensuse.org/issues/114881
+rm t/27-consoles-vmware.t
```

Comment:
please move it up into line 177

Reply:
I would rather keep it here, otherwise it becomes less obvious what ticket it relates to and future changes will cause conflicts

Reply:
We already reference tickets for individual excluded modules so we should be consistent